lib/group/searchGroups.js

```diff
@@ -0,0 +1,31 @@
+// Includes
+const getPageResults = require('../util/getPageResults.js').func
+
+// Args
+exports.required = ['keyword']
+exports.optional = ['prioritizeExactMatch', 'limit']
+
+// Docs
+/**
+ * ✅ Searches for groups by a given search term.
+ * @category Group
+ * @alias searchGroups
+ * @param {string} keyword - The keyword or search term to search by.
+ * @param {boolean} [true] prioritizeExactMatch - Whether or not to prioritize the exact match for the keyword
+ * @param {Limit=} [limit=∞] - The maximum number of groups to return
```

Comment:
What behavior does this manifest? Do most search terms go >10,000 results? Perhaps we should enforce an arbitrary default limit i.e. `100`, and let `false` correspond to infinity- otherwise we may see issues of people with infinite loops (in the practical sense)

Reply:
I agree; I'll admit I had not looked enough into the limiting behaviour as I was allowing prioritizeExactMatch to default to true.

I suggest a default limit of 100, but with the ability to supply `0` as no limit, so it's still of the same type.